barman/Dockerfile.template

```diff
@@ -6,7 +6,10 @@ RUN apt-get update \
 		rsync \
 		supervisor \
 	&& rm -Rf /var/lib/apt/lists/* \
-	&& sed -i 's/\(.*pam_loginuid.so\)/#\1/' /etc/pam.d/sshd
+	&& sed -i 's/\(.*pam_loginuid.so\)/#\1/' /etc/pam.d/sshd \
+	&& mkdir /home/postgres \
```

Comment:
I can do that if you like, but I think that, ordinarily, people consuming 
`tbeadle/postgres/Dockerfile-debian.template`
may be expecting `postgres` to be a nonuser-id, as the current template suggests, which makes sense for the "use postgres without barman" use case.

It is only because `tbeadle/postgres/barman` relies upon`/home/postgres/.ssh/authorized_keys` that creating and using `/home/postgres` makes sense to me.  I still can make the change that you requested instead, but "are you sure?"  :)